youtube_dl/extractor/bbc.py

```diff
@@ -1164,12 +1170,20 @@ def parse_media(media):
                         continue
                     formats, subtitles = self._download_media_selector(item_id)
                     self._sort_formats(formats)
+                    item_desc = try_get(media, lambda x: x['summary']['blocks'][0]['model']['text'], compat_str)
+                    item_time = None
+                    for meta in try_get(media, lambda x: x['metadata']['items'], list) or []:
+                        if try_get(meta, lambda x: x['label']) == 'Published':
+                            item_time = parse_iso8601(meta.get('timestamp'))
```

Comment:
`unified_timestamp`.

Reply:
Fine, a more general solution that I actually had originally but changed back, `parse_iso8601()` being already imported.